sections/csgcn/csgcn-chapter.tex

```diff
@@ -265,9 +265,9 @@ \subsection{Convolution layers}\label{subsec:csgcn_is_conv_layer}
 $N_{un}$ denotes the set of neighbors for the neighbor $un$ of node $u$, and $N_{in}$ denotes the set of neighbors for the neighbor $in$ of node $i$.
 This means that when neigbor nodes' representations are being propagated, the normalization is done with both the number of neighbors for the central node and the number of neighbors of the neighbor node that is currently having its reprentation propagated.
 \\\\
-The convolution layer propagates user and item nodes' representations by summing the normalized representations of the neighbor nodes.
-Only the item and user nodes' propagations are shown, as these embeddings are the only ones used beyond the convolution layer in the prediction function, further described in \Cref{subsec:csgcn_is_score_prediction}.
-The side-info nodes' representations are also being propagated with information about their neighbors in the layer, but are not being output from the layer since their representations are integrated into the item and user node representations, and therefore they are not used in the score prediction.
+The convolution layer propagates information to user and item nodes' representations by summing the normalized representations of the neighbor nodes.
+All types of nodes are being propagated with information about their neighbors, but since only the user and item nodes' represenations are being used in the prediciton function for our model, their representations are the only ones that are returned from the convolution layer.
+The argument for not also returning side-information nodes and using these in the prediction function, is that through the convolution, the information from these nodes are integrated into the representation of user and item nodes.
```

Comment:
```suggestion
The convolution layer propagates information to user and item nodes' representations by summing the normalized representations of the neighbor nodes.
All types of nodes are being propagated with information about their neighbors, but since only the user and item nodes' representations are being used in the prediction function for our model, their representations are the only ones that are returned from the convolution layer.
The argument for not also returning side-information nodes and using these in the prediction function is that through the convolution, the information from these nodes is integrated into the representation of user and item nodes.
```